tests/integration/cli/projects/hello-world/contracts/HelloWorld.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I'd like to add a deploy script as well as some tests as well as more features to the contract for this `HelloWorld` project, so it will be like a "real' project. This will come down the line as needed!